app/ui-react/packages/auto-form/src/widgets/TextButton.tsx

```diff
@@ -23,13 +26,15 @@ export class TextButton extends React.Component<ITextButtonProps> {
             onClick={this.props.onClick}
             className={`btn btn-link ${className}`}
             dangerouslySetInnerHTML={{ __html: this.props.linkText! }}
+            data-testid={id}
```

Comment:
With the previous suggestions, this can become `{...props}`, and all `this.props` can be dumped.